src/project-3.js

```diff
@@ -1,83 +1,93 @@
 // Do not change any of the function names
 
 const makeCat = (name, age) => {
-  // create a new object with a name property with the value set to the name argument
-  // add an age property to the object with the value set to the age argument
-  // add a method called meow that returns the string 'Meow!'
-  // return the object
+  const cat = {};
```

Comment:
This works but I think it looks a little better to write it like this:
```
return {
  name,
  age,
  meow: () => 'Meow!'
};
```
It's a little more concise and easier to read that everything is happening to the same object and that you're constructing a new object